src/commonMain/kotlin/com/charleskorn/kaml/Yaml.kt

```diff
@@ -80,8 +80,11 @@ public class Yaml(
 
     internal fun parseToYamlNode(source: Source): YamlNode {
         val parser = YamlParser(source, configuration.codePointLimit)
-        val reader =
-            YamlNodeReader(parser, configuration.extensionDefinitionPrefix, configuration.allowAnchorsAndAliases)
+        val reader = YamlNodeReader(
+            parser,
+            configuration.extensionDefinitionPrefix,
+            maxAliasCount = configuration.anchorsAndAliases.maxAliasCount,
```

Comment:
The explicit parameter name isn't necessary here.

```suggestion
            configuration.anchorsAndAliases.maxAliasCount,
```